### ISSUE TITLE
Example code does not match the library

### ISSUE THREAD

**Issue by HendrikRoth** [ankitdaf/Adafruit_Neopixel_ATTiny85#1]
Hello, your example code uses
```
pixels.setPixelColor(i, pixels.Color(255-i,abs(127-i),i));
```
but your function only accepts one parameter.

I tried to use this lib with an attiny45 but only the first (of 4) led lights up.

**Comment by ankitdaf** [comment]
Yes, I extended the library but I think I might have forgotten to commit it, will try to share it in a while, sorry for that !

**Comment by HendrikRoth** [comment]
Thank you for your response. I am not so great at c programming, so this would definitly help for my interactive led wall :-)
